### PR TITLE
Extended descriptions for terrain and furniture

### LINKED ISSUE #28497: Add extended description for more non-obvious tiles
**Is your feature request related to a problem? Please describe.**  
There are some tiles in the game that are not too self-explanatory and would thus benefit from an extended description. Many tiles are still lacking one, or the current extended description isn't that good so far.

**Describe the solution you'd like**  
Write an extended description for most tiles and describe how the “interactive” tiles can be used. Some tiles are a bit more complicated and are only useful in combination with other ti

The description should reflect:

* For interactive tiles: How they work and how they can be used (most important). Also, if it applies: how they interact with other tiles (example: fuel console)
* For machines and apparatuses that can't be used (like the arcade machine): Say in description that these are “broken”. Also maybe say what their pre-Cataclysm use was, in 1 sentence
* For furniture: (roughly) what they are made of (implications for deconstruction)
* For all tiles: What they are
* For all tiles: Whatever you write, write it spoiler-free

**High priority tiles**
- t_cvdmachine
- t_atm
- t_slot_machine (ext. description should explain the game, with all possible winnings and chances)
- t_gas_pump*
- t_diesel_pump*
- t_radio_controls
- t_marloss*
- t_radio_tower
- t_console
- t_gates_control*
- t_centrifuge
- t_manhole*
- t_card
- t_plut_generator

Tiles that might need an improvement:

* mailbox (“You've got mail.” is false)
* f_arcade_machine (cannot be used)
* f_pinball_machine (cannot be used)
* ball machine (cannot be used)
**Describe alternatives you've considered**  
None.

**Additional context**  
None.

### PR DESCRIPTION
**Pull request by Mooses2k**
#### Summary
```SUMMARY: Content "Extended descriptions for terrain and furniture"```

#### Purpose of change
Fixes #28497 

Previously, many furniture and terrain items didn't have useful descriptions that helped the player determine their purpose. I remedied this.

#### Describe the solution
I didn't extensively add a description for every single thing, but did many including all of the listed items in that issue and many more interactive tiles, thus I'd consider that it 'fixes' the issue.

As I found them, I also corrected typos, style issues, grammar, and tense. Tried to make things better than how I found them.

#### Describe alternatives you've considered
None.

#### Additional context
This is my first PR. Hope you enjoy the color. :)

### CODE REVIEW

**Comment by Mooses2k** [comment]
Appears to be a tabs and spaces issue. Resolving.

File: data/json/furniture.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
     "type": "furniture",
     "id": "f_rubble_landfill",
     "name": "pile of trashy rubble",
-    "description": "Trash topped with dirt and grass, it smells gross and but another mans trash...",
+    "description": "Trash topped with dirt and grass, it smells gross and but another man's trash...",
```
Comment (DSeyka) [inline data/json/furniture.json]:
```suggestion
    "description": "Trash topped with dirt and grass, it smells gross, but another man's trash...",
```

File: data/json/furniture.json [hunk 2]
```diff
@@ -283,7 +283,7 @@
     "type": "furniture",
     "id": "f_bulletin",
     "name": "bulletin board",
-    "description": "Pin some notes for other survivors to read.",
+    "description": "A big, cork bulletin board sporting various notices, most of them, depressing.  Pin some notes for other survivors to read.",
```
Comment (DSeyka) [inline data/json/furniture.json]:
```suggestion
    "description": "A big, cork bulletin board sporting various notices, most of them depressing.  Pin some notes for other survivors to read.",
```

File: data/json/furniture.json [hunk 3]
```diff
@@ -5878,7 +5910,7 @@
     "id": "f_server",
     "looks_like": "f_utility_shelf",
     "name": "server stack",
-    "description": "This is a big pile of computers.  They're all turned off.",
+    "description": "This is a big pile of computers.  They're all turned off...forever.",
```
Comment (DSeyka) [inline data/json/furniture.json]:
```suggestion
    "description": "This is a big pile of computers.  They're all turned off... forever.",
```

Comment (Mooses2k) [reply data/json/furniture.json]: For the 'They're all turned off...forever' - well, that's a style thing I've always used. Is there a common style choice across CDDA to have blah...[space]blah?

File: data/json/furniture.json [hunk 4]
```diff
@@ -835,7 +835,7 @@
     "id": "f_exercise",
     "name": "exercise machine",
     "symbol": "T",
-    "description": "Typically used for, well, exercising.  You're not up for it.",
+    "description": "Typically used for, well, exercising.  You're getting quite enough of that, running for your life.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]:
```suggestion
    "description": "Typically used for, well, exercising.  You're getting quite enough of that; running for your life.",
```

File: data/json/furniture.json [hunk 5]
```diff
@@ -866,7 +866,7 @@
     "type": "furniture",
     "id": "f_ball_mach",
     "name": "ball machine",
-    "description": "Remember when baseball was a thing?",
+    "description": "Remember when Skee-ball was a thing?  Well, the ball return doesn't seem to work without power and somebody's kicked in the coin slot.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]: I'm not actually sure what type of machine this is truly meant to represent, so you may want to look at places where it is referenced more in-depth to get a better idea of the description (something I had to do for my descriptions).

It could be a tennis ball machine, a ping pong ball machine, a skee ball machine or baseball pitching machine. If you can reword it to be a bit more generic:

```suggestion
    "description": "An unpowered machine that seems like it could've been used to launch various balls for different types of sports.  It's only good for parts now if disassembled.",
```

File: data/json/furniture.json [hunk 6]
```diff
@@ -1182,7 +1182,7 @@
     "id": "f_locker",
     "name": "locker",
     "symbol": "{",
-    "description": "Usually used for storing equipment or items.",
+    "description": "Used for storing equipment or items.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]: I think this is meant to reference being able to hide in lockers, but I don't think it was added correctly. Check pillow fort for the `HIDE_PLACE` flag, but that would need to be a separate (but tiny) PR.

File: data/json/furniture.json [hunk 7]
```diff
@@ -1309,7 +1309,7 @@
     "type": "furniture",
     "id": "f_dryer",
     "name": "dryer",
-    "description": "Dry your clothes!",
+    "description": "Dry your clothes!  Would be what you'd do if electricity was running.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]: You may want to add air quotes:

```suggestion
    "description": "'Dry your clothes!' would be what you'd do if electricity was running.",
```

File: data/json/furniture.json [hunk 8]
```diff
@@ -1544,7 +1544,7 @@
     "type": "furniture",
     "id": "f_crate_c",
     "name": "crate",
-    "description": "What's inside?  Find out!",
+    "description": "What's inside?  Break it or pry it open to find out!",
```
Comment (DracoGriffin) [inline data/json/furniture.json]: May want to reference that `break`ing crates can damage or destroy items inside. Or maybe that doesn't happen anymore, but it used to (if it doesn't anymore, then this comment is irrelevant!).

File: data/json/furniture.json [hunk 9]
```diff
@@ -2094,6 +2108,7 @@
     "type": "furniture",
     "id": "f_plant_harvest",
     "name": "harvestable plant",
+    "description": "This plant is ready for harvest.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]: You may want to imply how to harvest it.
```suggestion
    "description": "This plant is ready for harvest.  Examine it more closely to identify how to harvest the plant appropriately.",
```

File: data/json/furniture.json [hunk 10]
```diff
@@ -2408,7 +2423,7 @@
     "type": "furniture",
     "id": "f_mannequin",
     "name": "mannequin",
-    "description": "Put clothes on it and wish you looked as good.",
+    "description": "Put clothes on it, talk to it.  Who's around to judge you?  Wait...did it just move?",
```
Comment (DracoGriffin) [inline data/json/furniture.json]: [Spacing for ellipses is pure style](https://www.grammarbook.com/punctuation/ellipses.asp) but I'd go with whatever the majority has been used.

File: data/json/furniture.json [hunk 11]
```diff
@@ -2833,6 +2851,7 @@
     "type": "furniture",
     "id": "f_lilypad",
     "name": "lilypad",
+    "description": "The frogs around here aren't what they used to be.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]: Alternatively:

```suggestion
    "description": "These don't look like they could support the weight of the [amphibians/frogs] you've heard croaking in the swamps.",
```

File: data/json/furniture.json [hunk 12]
```diff
@@ -2896,6 +2915,7 @@
     "type": "furniture",
     "id": "f_egg_sackbw",
     "name": "spider egg sack",
+    "description": "Kind of icky.  Something IS moving in there.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]: If you feel compelled to be arachnophobia-inducing, you can try to describe the differences between the three egg sacs.

f_egg_sackbw is for black widows, f_egg_sackcs is for cellar spiders and f_egg_sackws is for wolf spiders (which generally are on or VERY close to the female wolf spider [they hatch onto her abdomen]).

[Black Widow w/ egg sac](https://ucanr.edu/blogs/bugsquad//blogfiles/24303_original.jpg).

[Cellar Spider w/ egg sac](https://spiderid.com/picture/1742/).

[Wolf Spider w/ egg sac](https://bugguide.net/node/view/143934).

File: data/json/furniture.json [hunk 13]
```diff
@@ -2968,7 +2991,7 @@
     "type": "furniture",
     "id": "f_arcade_machine",
     "name": "arcade machine",
-    "description": "Play stupid games, win stupid prizes.",
+    "description": "Play stupid games, win stupid prizes.  That was the idea, anyway.  Now, without power, it's just sad.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]:
```suggestion
    "description": "Play stupid games, win stupid prizes.  That was the idea, anyway.  Now, without power, it's just stupid.  Smarter to disassemble for all kinds of useful electronic parts.",
```

File: data/json/furniture.json [hunk 14]
```diff
@@ -3008,7 +3031,7 @@
     "type": "furniture",
     "id": "f_pinball_machine",
     "name": "pinball machine",
-    "description": "Most underrated game of the 20th century.  Press buttons so it doesn't go in the hole.",
+    "description": "Most underrated game of the 20th century.  Press buttons so the ball doesn't go in the hole.  It doesn't seem to be working without electricity.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]:
```suggestion
    "description": "Most underrated game of the 20th century.  Press buttons so the ball doesn't go in the hole.  It doesn't seem to be working without electricity.  Could be disassembled for various electronic parts.",
```

File: data/json/furniture.json [hunk 15]
```diff
@@ -3099,7 +3122,7 @@
     "type": "furniture",
     "id": "f_treadmill",
     "name": "treadmill",
-    "description": "Used for training leg muscles.  It'll be hard without power.",
+    "description": "Used for training leg muscles.  It'll be extra hard without power.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]:
```suggestion
    "description": "Used for training leg muscles.  It'll be extra hard without power.  Could be taken apart for its... parts.",
```

File: data/json/furniture.json [hunk 16]
```diff
@@ -3134,7 +3157,7 @@
     "type": "furniture",
     "id": "f_displaycase",
     "name": "display case",
-    "description": "Display your stuff.  Securely.",
+    "description": "Display your stuff securely.  Features fancy velvet couches cocked at a fetching angle.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]: Is there actual couches inside the case? I think this is a poor description, especially for translation issues.

File: data/json/furniture.json [hunk 17]
```diff
@@ -3326,6 +3349,7 @@
     "type": "furniture",
     "id": "f_robotic_arm",
     "name": "robotic arm",
+    "description": "Automation!  Science!  Industry!  Make a better horse!  This robot arm promises to do it all.  Except it's currently unpowered.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]:
```suggestion
    "description": "Automation!  Science!  Industry!  Make a better horse!  This robot arm promises to do it all.  Except it's currently unpowered.  You could remove the casing and retrieve the electronics through disassembly.",
```

File: data/json/furniture.json [hunk 18]
```diff
@@ -4236,7 +4268,7 @@
     "type": "furniture",
     "id": "f_piano",
     "name": "piano",
-    "description": "The ol' ebony and ivory.",
+    "description": "The ol' ebony and ivory.  Really classes up the place.",
```
Comment (DracoGriffin) [inline data/json/furniture.json]:
```suggestion
    "description": "The ol' ebony and ivory.  Really classes up the place.  You could take it apart if you wanted... you monster.",
```

File: data/json/terrain.json [hunk 19]
```diff
@@ -7055,6 +7060,7 @@
     "type": "terrain",
     "id": "t_radio_tower",
     "name": "radio tower",
+    "description": "This the structure of a radio transmission tower.",
```
Comment (DracoGriffin) [inline data/json/terrain.json]:
```suggestion
    "description": "This is the structure [of/for] a radio transmission tower.",
```

File: data/json/terrain.json [hunk 20]
```diff
@@ -7076,6 +7082,7 @@
     "type": "terrain",
     "id": "t_radio_controls",
     "name": "radio controls",
+    "description": "This the console controlling a radio transmission tower.  It doesn't seem to be fully operational.",
```
Comment (DracoGriffin) [inline data/json/terrain.json]:
```suggestion
    "description": "This console appears to control a nearby radio transmission tower.  It doesn't seem to be fully operational.",
```

File: data/json/terrain.json [hunk 21]
```diff
@@ -7440,6 +7459,7 @@
     "type": "terrain",
     "id": "t_centrifuge",
     "name": "centrifuge",
+    "description": "This is centrifuge, a liquid separating device with an automated analyzer unit. It could be used to analyze a medical fluid sample, such as blood, if a test tube was placed in it.",
```
Comment (DracoGriffin) [inline data/json/terrain.json]:
```suggestion
    "description": "This is a centrifuge; a liquid separating device with an automated analyzer unit. It could be used to analyze a medical fluid sample, such as blood, if a test tube was placed in it.",
```

File: data/json/terrain.json [hunk 22]
```diff
@@ -7465,6 +7485,7 @@
     "type": "terrain",
     "id": "t_cvdbody",
     "name": "CVD machine",
+    "description": "This the bulk of a highly technical-looking apparatus controlled by a nearby console.",
```
Comment (DracoGriffin) [inline data/json/terrain.json]:
```suggestion
    "description": "The bulk of a highly technical-looking apparatus controlled by a nearby console.",
```

File: data/json/terrain.json [hunk 23]
```diff
@@ -7777,6 +7803,7 @@
     "type": "terrain",
     "id": "t_card_reader_broken",
     "name": "broken card reader",
+    "description": "This is a smartcard reader, but it doesn't seem to be functioning.",
```
Comment (DracoGriffin) [inline data/json/terrain.json]:
```suggestion
    "description": "This is a smartcard reader, but it doesn't seem to be functioning.  Probably because there's no more blinking red LED.",
```

File: data/json/furniture.json (hunk 4 above)
Comment (Mooses2k) [reply data/json/furniture.json]: > 

Technically, "running for your life" is a partial phrase (I'm blanking on what it's actually called), that's dependent on the previous part of the sentence. It doesn't stand on its own.

File: data/json/furniture.json (hunk 5 above)
Comment (Mooses2k) [reply data/json/furniture.json]: > 
> 
> I'm not actually sure what type of machine this is truly meant to represent, so you may want to look at places where it is referenced more in-depth to get a better idea of the description (something I had to do for my descriptions).
> 
> It could be a tennis ball machine, a ping pong ball machine, a skee ball machine or baseball pitching machine. If you can reword it to be a bit more generic:

We talked about it on Discord and decided Skee-ball specifically. I can make it more generic though, if you like. And definitely add the disassembly bit.

File: data/json/furniture.json (hunk 6 above)
Comment (Mooses2k) [reply data/json/furniture.json]: > 
> 
> I think this is meant to reference being able to hide in lockers, but I don't think it was added correctly. Check pillow fort for the `HIDE_PLACE` flag, but that would need to be a separate (but tiny) PR.

Got it. I'll revert that.

File: data/json/furniture.json (hunk 3 above)
Comment (Night-Pryanik) [reply data/json/furniture.json]: Punctuation marks are separated from the following words.

File: data/json/furniture.json (hunk 8 above)
Comment (Mooses2k) [reply data/json/furniture.json]: Or maybe we want the player to learn that the hard way? Since it could be considered obvious?

File: data/json/furniture.json (hunk 10 above)
Comment (Mooses2k) [reply data/json/furniture.json]: Totally on board with following style...I just don't know what it is?

File: data/json/furniture.json (hunk 11 above)
Comment (Mooses2k) [reply data/json/furniture.json]: Done.

File: data/json/furniture.json (hunk 16 above)
Comment (Mooses2k) [reply data/json/furniture.json]: Agreed. Shouldn't add things to the description that require new disassembly items.

File: data/json/furniture.json (hunk 12 above)
Comment (Mooses2k) [reply data/json/furniture.json]: Done

File: data/json/furniture.json (hunk 8 above)
Comment (DracoGriffin) [reply data/json/furniture.json]: Personally, if players are reading the descriptions, they *should* be rewarded for taking that effort. Meta-wise, it is fairly easy for players to discover out of game, no reason to punish them for it.

File: data/json/furniture.json (hunk 4 above)
Comment (DracoGriffin) [reply data/json/furniture.json]: Both are complete phrases, and checking [this source](https://www.grammarly.com/blog/semicolon/) seems like an accurate use.

What you are doing is called a "[comma splice](https://en.wikipedia.org/wiki/Comma_splice)", which can be both an error or informal writing. However, in consideration for translators (which may not be an issue but I tend to err on caution as I *don't* speak [non-SVO languages](https://en.wikipedia.org/wiki/Subject%E2%80%93verb%E2%80%93object)), it's typically better to avoid too much slang/informal writing as it can be difficult to convey in other languages accurately.

File: data/json/furniture.json [hunk 24]
```diff
@@ -283,7 +283,7 @@
     "type": "furniture",
     "id": "f_bulletin",
     "name": "bulletin board",
-    "description": "Pin some notes for other survivors to read.",
+    "description": "A big, cork bulletin board sporting various notices, most of them depressing.  Pin some notes for other survivors to read.",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: The primary use of a bulletin board is as a camp message board, so you probably don't want such a specific description.

File: data/json/furniture.json [hunk 25]
```diff
@@ -461,7 +461,7 @@
     "id": "f_straw_bed",
     "name": "straw bed",
     "symbol": "#",
-    "description": "Kinda itches when you lay on it.",
+    "description": "A straw bed.  Kinda itches when you lay on it.",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: This seems a bit redundant, since the name field says it's a straw bes

File: data/json/furniture.json [hunk 26]
```diff
@@ -119,7 +119,7 @@
     "type": "furniture",
     "id": "f_wreckage",
     "name": "metal wreckage",
-    "description": "Pile of various bent and twisted metals.",
+    "description": "Pile of various bent and twisted metals.  Looks dangerous to walk on.",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: I think you should avoid lateral changes, this means pretty much the same thing

File: data/json/furniture.json [hunk 27]
```diff
@@ -412,7 +412,7 @@
     "name": "toilet",
     "symbol": "&",
     "color": "white",
-    "description": "Emergency water source, from the tank, and provider of relief.",
+    "description": "A porcelain throne.  Emergency water source (only from the tank) and provider of relief.",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: I think the original comma clause was more accurate and natural sounding.

File: data/json/furniture.json [hunk 28]
```diff
@@ -488,7 +488,7 @@
     "id": "f_sink",
     "name": "sink",
     "symbol": "&",
-    "description": "Emergency relief provider.  Water isn't running, so no water.",
+    "description": "A sink basin.  If only you could wash your hands of what you have done.  Water isn't running, so it's basically useless.",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: The sink includes a cabinet, not just a basin. I think you should avoid roleplaying comments here; not everyone plays a survivor that feels like they need to wash their hands of what they've done

File: data/json/furniture.json (hunk 10 above)
Comment (I-am-Erk) [reply data/json/furniture.json]: `wait... Did it just move?` would be the standard

File: data/json/furniture.json (hunk 3 above)
Comment (I-am-Erk) [reply data/json/furniture.json]: I'd about the whole "... forever" thing anyway; I don't think it adds anything to the description, and as powered furnishings are added it may become outdated.

**Comment by Mooses2k** [comment]
Any other suggestions? I think the Travis build failed due to some server degradation on their end? Want to re-run it or have me make any other changes?

**Comment by I-am-Erk** [comment]
There are a lot of these that are just minor rewordings to change it to the way you'd have said it, without changing the clarity or amount of information conveyed. I am really not a fan of that, personally... It's just removing another contributor's writing for the sake of removing it.

File: data/json/furniture.json [hunk 29]
```diff
@@ -1477,7 +1477,7 @@
     "type": "furniture",
     "id": "f_coffin_c",
     "name": "coffin",
-    "description": "Holds the bodies of the countless you kill.",
+    "description": "Holds the bodies of the countless killed in the Cataclysm.  The Zed, however, don't get buried.  They get pulped.",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: Technically you could bury a pulped zombie if you wanted, and there may be times you want to. I would take the second sentence out.

File: data/json/furniture.json [hunk 30]
```diff
@@ -1576,7 +1576,7 @@
     "type": "furniture",
     "id": "f_crate_o",
     "name": "open crate",
-    "description": "What's inside?  Look in it!",
+    "description": "What's in the box?  Look in it!",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: This is one of those lateral changes just for the sake of change.

File: data/json/furniture.json [hunk 31]
```diff
@@ -431,7 +431,7 @@
     "id": "f_makeshift_bed",
     "name": "makeshift bed",
     "symbol": "#",
-    "description": "Not as comfortable as a real bed, but it will suffice.",
+    "description": "Not as comfortable as a real bed, but it'll do in a pinch.",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: This is one of those lateral changes just for the sake of change.

File: data/json/furniture.json [hunk 32]
```diff
@@ -722,7 +722,7 @@
     "id": "f_sofa",
     "name": "sofa",
     "symbol": "H",
-    "description": "Lie down OR sit down!  Perfect!",
+    "description": "Lie down OR sit down!  What will they think of next!?",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: This is one of those lateral changes just for the sake of change.

File: data/json/furniture.json [hunk 33]
```diff
@@ -1349,7 +1349,7 @@
     "id": "f_bigmirror",
     "name": "standing mirror",
     "symbol": "{",
-    "description": "Lookin' good- is that blood?",
+    "description": "Lookin' sharp!  Wait, is that blood?",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: This is one of those lateral changes just for the sake of change.

File: data/json/furniture.json [hunk 34]
```diff
@@ -2052,6 +2063,7 @@
     "type": "furniture",
     "id": "f_plant_seed",
     "name": "seed",
+    "description": "A humble planted seed. Actions are the seed of fate deeds grow into destiny.",
     "symbol": "^",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: Missing a double space after `seed.`

File: data/json/furniture.json [hunk 35]
```diff
@@ -5998,7 +6030,7 @@
     "id": "f_dishwasher",
     "name": "dishwasher",
     "looks_like": "f_oven",
-    "description": "This metal box used to spray hot water and soap at dirty dishes to make them clean and to save people an unpleasant chore.  Now, with the power off and it sitting for a while, it is starting to smell a bit off.",
+    "description": "This metal box used to spray hot water and soap at dirty dishes to make them clean and to save people an unpleasant chore.  Now, with the power gone and it sitting for a while, it's starting to smell a bit off.",
```
Comment (I-am-Erk) [inline data/json/furniture.json]: This is one of those lateral changes just for the sake of change.

File: data/json/terrain.json [hunk 36]
```diff
@@ -7400,6 +7417,7 @@
     "type": "terrain",
     "id": "t_sewage_pipe",
     "name": "sewage pipe",
+    "description": "This fine specimen is a corridor through which travels the excrement of society.",
```
Comment (I-am-Erk) [inline data/json/terrain.json]: Sewage pipes are currently used in a lot of random places as decorative terrain. I'd suggest changing it to just being a large gauge pipe, and maybe changing the name too.

File: data/json/furniture.json (hunk 29 above)
Comment (Mooses2k) [reply data/json/furniture.json]: I mainly didn't like how it suggests that I bury zombie bodies...which I've never done. Perhaps just "Holds the bodies of the countless killed in the Cataclysm."

File: data/json/furniture.json (hunk 30 above)
Comment (Mooses2k) [reply data/json/furniture.json]: understood

File: data/json/furniture.json (hunk 35 above)
Comment (Mooses2k) [reply data/json/furniture.json]: No, this was for flow. Reuses the word 'off' in two different contexts and sounds wrong to me.

File: data/json/terrain.json (hunk 36 above)
Comment (Mooses2k) [reply data/json/terrain.json]: k

File: data/json/furniture.json (hunk 33 above)
Comment (Mooses2k) [reply data/json/furniture.json]: Changed due to that hyphen looking weird. Should at least be a space on either side. I'll do that, then, instead.

**Comment by Adamant36** [comment]
There's a lot that could be improved even in parts that add detail. I added some comments, but I didn't want to add to many. Maybe I'll do a PR myself building on this after it is merged. I've been wanting to contribute some grammatical fixes for a while now. I can just add more comments about what I think could improved in this one instead though if it would be better. Either way.

**Comment by kevingranade** [comment]
Notes on an existing PR are better than waiting and opening a new PR.

**Comment by Mooses2k** [comment]
Rebuild Gorgon

**Comment by anothersimulacrum** [comment]
Jenkins rebuild

**Comment by Mooses2k** [comment]
Shall I attempt to successfully-this-time rebase this?

**Comment by ZhilkinSerg** [comment]
> 
> 
> Shall I attempt to successfully-this-time rebase this?

Nah, I'll just squash it.
